.changeset/four-drinks-yell.md

```diff
@@ -0,0 +1,7 @@
+---
+'@aws-amplify/integration-tests': minor
+'@aws-amplify/backend-storage': minor
+'@aws-amplify/plugin-types': minor
+---
+
+add new type to handle exposing stack, expose stack in storage
```

Comment:
We can remove integration-tests from changesets.
I'd move storage from this changeset to the other one. and keep this one for plugin types only.
It'll generate more coherent changelogs.